index.js

```diff
@@ -0,0 +1,165 @@
+"use strict";
+//Get days in year
+const getDaysInYear = (year) => {
+  let date = new Date(year, 2);
+  let numberDay = 0;
+  date.setDate(0);
+  const n = date.getDate();
+  n === 28 ? (numberDay = 365) : (numberDay = 366);
+  document.getElementById("outputCountDay").value = numberDay;
+};
+
+//Get day of a year
+const getDayNumber = (date) => {
+  let newDate = new Date(date);
+  const year = newDate.getFullYear();
+  const month = newDate.getMonth();
+  const day = newDate.getDate();
+  let numberDay = 0;
+  for (let i = 0; i < month; i++) {
+    let newDate = new Date(year, i + 1, 0);
```

Comment:
1. please use const 
2. please create a unique name for this var (it's not a good practice to have vars with identical name)